modules/logging-multi-clf-enabling-additional-deployments.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/logging-multi-clf.adoc
+
+:_content-type: PROCEDURE
+[id="logging-multi-clf-enabling-additional-deployments_{context}"]
+= Enabling multiple log forwarder mode
+
+Cluster administrators must explicitly enable multiple log forwarder mode in the cluster by configuring RBAC permissions.
+
+.Prerequisites
+
+* The Cluster Logging Operator is installed in the `openshift-logging` namespace.
+
+.Procedure
+
+. Create a service account for the collector. If you want to write logs to storage that requires a token for authentication, you must include a token in the service account.
+
+. Create cluster roles to define which log types each role is allowed to collect:
++
+.ClusterRole resource example
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: collect-audit-logs
+rules:
+- apiGroups:
+  - "logging.openshift.io"
+  resources:
+  - logs
+  resourceNames:
+  - audit
+  verbs:
+  - collect
+# ...
+----
+
+. Bind each cluster role to the service account:
++
+.Example binding command
+[source,terminal]
+----
+$ oc create clusterrolebinding <cluster_role_binding_name> --clusterrole=<cluster_role_name> --serviceaccount=<namespace:service_account_name>
+----
+
+. Create a `ClusterLogForwarder` custom resource (CR) that references the service account and specifies the log input types that the service account can collect:
++
+.ClusterLogForwarder resource example
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: audit-collector
+  namespace: openshift-kube-apiserver
+spec:
+  serviceAccount: audit-collector-sa <1>
+  pipelines:
+   - inputRefs:
+     - audit <2>
+     outputRefs:
+     - loki
+  outputs:
+  - name: loki
+    type: loki
+    url: https://mycenteralizedserver.some.place
+# ...
+----
+<1> The name of the service account.
+<2> The log types that can be collected. In this case, the log input type specifies audit logs.
+
+. Optional. If you want to specify collector resources or deployments, you must create a `ClusterLogging` CR:
++
+.ClusterLogging resource example
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: audit-collector <1>
+  namespace: openshift-kube-apiserver <2>
+spec:
+  collection:
+    type: "vector" <3>
```

Comment:
since we mention modifying the collector resources maybe we include it in the example ie:
```
type: vector
resources:
  limits:
    memory: 2G
```

Reply:
I think this is something for a future PR now that I see there are ClusterLogging CR docs already. Those need to be cleaned up, but I think it's out of scope for the multi-CLF user story perhaps?